compiler/ballerina-parser/src/main/resources/syntax_diagnostic_message.properties

```diff
@@ -202,6 +215,8 @@ error.import.declaration.after.other.declarations=import declaration after other
 error.annotations.attached.to.expression=annotations attached to expression
 error.invalid.expression.in.start.action=invalid expression in start action
 error.same.object.type.qualifier=same object type qualifier
+error.duplicate.qualifier=duplicate qualifier
+error.qualifier.not.allowed=qualifier not allowed
```

Comment:
I think error msg like this:  `''{0}'' qualifier not allowed here` would be better. 
In the class-change branch it is already using that way for `error.qualifier.not.allowed`